Signal/src/ViewControllers/ContactsViewHelper.m

```diff
@@ -26,19 +26,26 @@ @interface ContactsViewHelper ()
 
 @property (nonatomic) NSArray<NSString *> *blockedPhoneNumbers;
 
+@property (nonatomic) BOOL suppressDelegateEvents;
+
 @end
 
 #pragma mark -
 
 @implementation ContactsViewHelper
 
-- (instancetype)init
+- (instancetype)initWithDelegate:(id<ContactsViewHelperDelegate>)delegate
 {
     self = [super init];
     if (!self) {
         return self;
     }
 
+    OWSAssert(delegate);
+    _delegate = delegate;
+
+    self.suppressDelegateEvents = YES;
```

Comment:
Ah, sorry, with the GH split diff I missed the call to `[self updateContacts];` which uses the shouldHideLocalNumber (via another method call).

This makes sense, but a comment would still be helpful.